src/guacd/daemon.c

```diff
@@ -504,6 +555,26 @@ int main(int argc, char* argv[]) {
 
     }
 
+    /* Stop all connections */
+    if (map != NULL) {
+
+        guacd_proc_map_foreach(map, stop_process_callback, NULL);
+
+        /*
+         * FIXME: Clean up the proc map. This is not as straightforward as it
```

Comment:
The daemon was never really set up to perform cleanup, and it's less easy to get everything cleaned up than I had hoped. In the service of avoiding further complexity ballooning, I'm going to leave this for future improvement.

The map was never cleaned up before either, so at least this isn't making things worse than it used to be.